### PR TITLE
add regex for known installation issues

### PR DESCRIPTION
**Pull request by bmeng**
add the known installation issue for [OSD-9693](https://issues.redhat.com/browse/OSD-9693) [OSD-9695](https://issues.redhat.com/browse/OSD-9695) and [OSD-9694](https://issues.redhat.com/browse/OSD-9694)

### CODE REVIEW

**Comment by gregsheremeta** [comment]
partially duplicated by @karthikperu7 in #1661

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 1]
```diff
@@ -149,6 +149,24 @@ data:
       - "The vpc ID .* does not exist"
       installFailingReason: AWSVPCDoesNotExist
       installFailingMessage: The AWS VPC does not exist
+    - name: VPCDhcpOptionAssociationAbsent
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2032521
+      searchRegexStrings:
+      - "error .* applying changes to module.vpc.aws_vpc_dhcp_options_association"
```
[the remark this answers is not in the document]
Comment (2uasimojo) [reply config/configmaps/install-log-regexes-configmap.yaml:155]: If the concern is those chunks being on different lines of the output, you can use `(?s)` in the regex, like [AWSNoWorkerNodes](https://github.com/openshift/hive/blob/9aec08e07074c8b8252f5c62e8033f2919ec5717/config/configmaps/install-log-regexes-configmap.yaml#L107)

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 2]
```diff
@@ -149,6 +149,24 @@ data:
...
+    - name: TargetGroupNotFound
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1898265
+      searchRegexStrings:
+      - "TargetGroupNotFound"
+      installFailingReason: TargetGroupNotFound
+      installFailingMessage: Target Group cannot be found after creation
+    - name: RouteTablePrivateRoutesAbsent
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2033256
+      searchRegexStrings:
+      - "ERROR When applying changes to module.vpc.aws_route_table.private_routes"
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:167]: Same comment here about adding the "was present, but now absent" piece to the regex.

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 3]
```diff
@@ -149,6 +149,24 @@ data:
       - "The vpc ID .* does not exist"
       installFailingReason: AWSVPCDoesNotExist
       installFailingMessage: The AWS VPC does not exist
+    - name: VPCDhcpOptionAssociationAbsent
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:152]:
```suggestion
    - name: VPCDhcpOptionAssociationPresentNowAbsent
```

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 4]
```diff
@@ -149,6 +149,24 @@ data:
...
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2032521
+      searchRegexStrings:
+      - "error .* applying changes to module.vpc.aws_vpc_dhcp_options_association"
+      installFailingReason: VPCDhcpOptionAssociationAbsent
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:156]:
```suggestion
      installFailingReason: VPCDhcpOptionAssociationPresentNowAbsent
```

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 5]
```diff
@@ -149,6 +149,24 @@ data:
...
+      installFailingMessage: The VPC Dhcp Option Association absent after creation
+    - name: TargetGroupNotFound
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1898265
+      searchRegexStrings:
+      - "TargetGroupNotFound"
+      installFailingReason: TargetGroupNotFound
+      installFailingMessage: Target Group cannot be found after creation
+    - name: RouteTablePrivateRoutesAbsent
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:164]:
```suggestion
    - name: RouteTablePrivateRoutesPresentNowAbsent
```

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 6]
```diff
@@ -149,6 +149,24 @@ data:
...
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2033256
+      searchRegexStrings:
+      - "ERROR When applying changes to module.vpc.aws_route_table.private_routes"
+      installFailingReason: RouteTablePrivateRoutesAbsent
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:168]:
```suggestion
      installFailingReason: RouteTablePrivateRoutesPresentNowAbsent
```

**Pull request merged**